Corona-Warn-App/src/main/res/layout/include_risk_card_content.xml

```diff
@@ -1,281 +1,287 @@
-                android:accessibilityHeading="true"
+                android:id="@+id/risk_card_progress_headline"
                 style="@style/headline5"
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:layout_marginEnd="@dimen/spacing_normal"
-                android:text="@{tracingCard.getRiskLevelHeadline(context)}"
-                android:textColor="@{tracingCard.getStableTextColor(context)}"
-                app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintEnd_toStartOf="@+id/risk_card_header_icon"
+                android:layout_marginEnd="16dp"
+                android:accessibilityHeading="true"
+                android:text="@{tracingCard.getProgressCardHeadline(context)}"
+                android:textColor="@color/colorStableLight"
+                app:layout_constraintEnd_toStartOf="@+id/risk_card_progress_headline_icon"
                 app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent" />
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_goneMarginEnd="0dp"
+                tools:text="Daten werden herunter geladen" />
 
             <ImageView
-                android:id="@+id/risk_card_header_icon"
+                android:id="@+id/risk_card_progress_headline_icon"
+                gone="@{tracingCard.showDetails}"
                 android:layout_width="@dimen/icon_size_risk_card"
                 android:layout_height="@dimen/icon_size_risk_card"
                 android:importantForAccessibility="no"
                 android:src="@drawable/ic_forward"
-                android:tint="@{tracingCard.getStableTextColor(context)}"
-                gone="@{tracingCard.showDetails}"
+                android:tint="@color/colorStableLight"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintTop_toTopOf="parent" />
 
-        </androidx.constraintlayout.widget.ConstraintLayout>
-
-        <!-- Loading State -->
-        <androidx.constraintlayout.widget.ConstraintLayout
-            android:layout_width="@dimen/match_constraint"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/spacing_normal"
-            gone="@{!tracingCard.refreshing}"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/risk_card_header">
-
             <ProgressBar
-                android:id="@+id/risk_card_progress"
+                android:id="@+id/risk_card_progress_indicator"
                 style="@style/progressBar"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_marginStart="@dimen/spacing_small"
+                android:layout_width="36dp"
```

Comment:
also here use one of the given dimens, we don't have `32dp`. So I would use `spacing_medium`

Reply:
That does not work, it's not about using a predefined spacing, but that the UI looks right, which requires this to be 36, or does it not look correct in the UI in this branch?

Reply:
than maybe add this as a new dimen as we have it for the "circles". We normally never have dimen directly in the layout file.

Reply:
Not about spacing but the size of the progress indicator. Please check the current size and then let me know whether you want it bigger or smaller just to use one of the spacing dimensions.

I personally don't like the hardcoded spacing in things that are unlikely to be changed, or not in places where they are not repeated in other elements. In this case it's the specific size of a unique element.

Reply:
It's more obvious, at least to me, how something looks based on seeing the actual `dp` values then guessing "what was `spacing_tiny` again? :shrug: